inference-engine/src/mkldnn_plugin/nodes/mkldnn_region_yolo_node.cpp

```diff
@@ -367,13 +367,13 @@ inline void MKLDNNRegionYoloNode::calculate_logistic(size_t start_index, int cou
 }
 
 void MKLDNNRegionYoloNode::execute(mkldnn::stream strm) {
-    auto inputDesc = getParentEdgeAt(0)->getDesc();
-    auto outputDesc = getChildEdgeAt(0)->getDesc();
+    // auto inputDesc = getParentEdgeAt(0)->getMemory().GetDesc().
+    // auto outputDesc = (0)->getDesc();
```

Comment:
Please either provide the reason why the code was commented or remove these lines.

Reply:
removed